app/assets/javascripts/admin/order_cycles/controllers/order_cycles_controller.js.coffee

```diff
@@ -15,7 +15,9 @@ angular.module("admin.orderCycles").controller "OrderCyclesCtrl", ($scope, $q, C
       Dereferencer.dereference(schedule.order_cycles, OrderCycles.byID)
     for orderCycle in $scope.orderCycles
       coordinator = Enterprises.byID[orderCycle.coordinator.id]
-      orderCycle.coordinator = coordinator if coordinator?
+      if coordinator?
+        orderCycle.coordinator = coordinator
+        orderCycle.coordinator.name = $sce.trustAsHtml(coordinator.name)
```

Comment:
The unit tests fail, because `$sce` is unknown. You need to add it as dependency to make this work.

Reply:
I'm not being able to execute the tests locally:

![image](https://user-images.githubusercontent.com/7442967/52500238-a860a080-2bc4-11e9-9f91-227e86ce5c31.png)

Reply:
You may not need fsevents to tun the tests. Have you tried to execute them?

Other options include:
- Set up a container to run the tests.
- Set up Semaphore CI on your fork to run the tests.
- Just push to this branch so that our tests get run.

One spec is still failing: `spec/javascripts/unit/admin/order_cycles/controllers/order_cycles_controller_spec.js:53`

I think the problem is not your code, it's the spec that needs updating. It doesn't know about the new name entry.

Reply:
AFAIK fsevents is not available in windows. Is that your case @laka3000 ?

Reply:
@sauloperez I'm or Linux Mint rn, but I'll try the steps @mkllnk provided